enterprise/core-edge/src/main/java/org/neo4j/coreedge/server/CoreEdgeClusterSettings.java

```diff
@@ -94,12 +94,14 @@ public String toString()
             setting( "core_edge.log_shipping_max_lag", INTEGER, "256" );
 
     @Description("Size of the RAFT in queue")
+    @Internal
     public static final Setting<Integer> raft_in_queue_size =
             setting( "core_edge.raft_in_queue_size", INTEGER, "64" );
 
     @Description("Largest batch processed by RAFT")
+    @Internal
     public static final Setting<Integer> raft_in_queue_max_batch =
-            setting( "core_edge.raft_in_queue_max_batch", INTEGER, "16" );
+            setting( "core_edge.raft_in_queue_max_batch", INTEGER, "64" );
```

Comment:
Is  "core_edge.raft_in_queue_max_batch" related to "core_edge.raft_in_queue_size"? If so maybe we should link them somehow.